week1/homework/src/server.js

```diff
@@ -1,20 +1,29 @@
+/* eslint-disable indent */
 'use strict';
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
-function createServer(port) {
+const createServer = () => {
   let state = 10;
-
-  const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+  const server = http.createServer((req, res) => {
+    const RenderResponse = (statusCode, write) => {
```

Comment:
JavaScript variables are in `camelCase`
Also `write` doesn't look the best name for the data will be sent.

Reply:
Done!